### DISCUSSION TITLE
工具功能区打不开

### DISCUSSION THREAD

**Discussion by zcaiang** [he3-app/he3-feedback#263] [Bug Report]
选择某个工具后，打不开该工具，界面一片空白。
![image](https://github.com/he3-app/he3-feedback/assets/46180563/7179f9f8-a32e-4e2c-8043-65a58d62e871)

Edition	Windows 10 Enterprise
Version	22H2
Installed on	‎2022/‎8/‎8
OS build	19045.3324
Experience	Windows Feature Experience Pack 1000.19041.1000.0

**Comment by lyzhang1999** [comment]
请检查网络对以下域名的访问情况：

api.he3app.com
cdn.he3app.com
portal.he3app.com
t.he3app.com
docs.he3app.com
webview.he3app.com
